packages/grafana-ui/src/components/Legend/LegendSeriesIcon.tsx

```diff
@@ -1,21 +1,38 @@
 import React from 'react';
 import { SeriesColorPicker } from '../ColorPicker/ColorPicker';
-import { SeriesIcon } from './SeriesIcon';
+import { SeriesIcon, SeriesIconProps } from './SeriesIcon';
 
 interface LegendSeriesIconProps {
+  disabled: boolean;
```

Comment:
what does a disabled icon mean?

Reply:
It's an icon that's not clickable, meaning a one without `SeriesColorPicker`

Reply:
I guess this could be named `LegendSeriesButton`, but on the other hand it ain't a button.